CIP-0030/README.md

```diff
@@ -202,6 +218,12 @@ Errors: `APIError`
 
 Returns the network id of the currently connected account. 0 is testnet and 1 is mainnet but other networks can possibly be returned by wallets. Those other network ID values are not governed by this document. This result will stay the same unless the connected account has changed.
 
+### api.getChainId(): Promise\<chain_id>
```

Comment:
For the sake of consistency with the `supportedChains` API property, I guess this should rather be
```suggestion
### api.getChainId(): Promise\<cbor\<chain_id>>
```